### ISSUE TITLE
Can a still image drive be used to get a video?

### ISSUE THREAD

**Issue by zhanghongyong123456** [guoyww/AnimateDiff#21]

**Comment by huxiuhan** [comment]
See our quick hack on this: https://github.com/talesofai/AnimateDiff

**Comment by zhanghongyong123456** [comment]
> See our quick hack on this: https://github.com/talesofai/AnimateDiff
How can the original image be edited without changing，

I test based on the initial image generation, the generated image will change the content

**Comment by LiuShiyu95** [comment]
> > See our quick hack on this: https://github.com/talesofai/AnimateDiff
> > How can the original image be edited without changing，
> 
> I test based on the initial image generation, the generated image will change the content

Hello, could you please provide an example? Thank you！

**Comment by Bendito999** [comment]
For some reason on mine when I try to use other models the initial image is very blurry like just a blob, but the built in example of 10-InitImageYoimiya does work on mine and produces results similar to the ones they posted. Doing further experiments to see if I can get good results for other initial pictures.  I do wonder why it is so blurry sometimes, if there is a way to adjust the blur applied to the initial image.

**Comment by huxiuhan** [comment]
> For some reason on mine when I try to use other models the initial image is very blurry like just a blob, but the built in example of 10-InitImageYoimiya does work on mine and produces results similar to the ones they posted. Doing further experiments to see if I can get good results for other initial pictures. I do wonder why it is so blurry sometimes, if there is a way to adjust the blur applied to the initial image.
> 
> Edit: I do get some better results if, when trying to 'animate' an image that was originally made in Stable Diffusion prior at least, if I use the custom dreambooth model that originally made the image, set that up as the model that Animate uses (have base set to "" and "path" set to the dreambooth model, may try it the other way around to use the dreambooth model as 'base', unclear what the difference might be). I also set the seed and prompt the same in the .yaml, then set the init image to the original image that Stable Diffusion made (otherwise the picture it makes is way different than the stable diffusion picture because it isn't using Euler-A or whatever). Using the original image as the init, and using (roughly) the same prompt and seed settings in AnimateDiff that were used to make the original image, makes a recognizable result at least, instead of a blob.

Yes the init image from my implementation has very strong limitations. It works nicely with images that generated with similar lora + foundation model settings, but not generalize very well in many other cases.
But people can try to play out with some other parameter settings, especially on the way inserting initial latents. Hope community can help on this and move forward together. :)
